railties/lib/rails/generators/rails/plugin/templates/bin/test.tt

```diff
@@ -5,6 +5,6 @@ require 'rails/test_unit/minitest_plugin'
 
 Rails::TestUnitReporter.executable = 'bin/test'
 
-Minitest.run_via[:rails] = true
+Minitest.run_via = :rails
```

Comment:
Currently there is no update task for plugin. Also, plugin currently does not load the task provided by rails. Therefore, if provide a task similar to `rails app:update` for plugin, first need to update the plugin's Rakefile.   
However, it is inconvenient that there is no task for updating, so I will add it with another PR.